packages/server/lib/middleware/access.middleware.ts

```diff
@@ -255,8 +234,9 @@ export class AccessMiddleware {
                 connectSessionId: String(result.value.connectSession.id)
             });
             next();
-        } catch (e) {
-            logger.error(`failed_get_env_by_connect_session ${stringifyError(e)}`);
+        } catch (err) {
```

Comment:
renaming to `err` for consistency with the rest of the file

Reply:
It reminds me I need to load a eslint rule for that